### ISSUE TITLE
(match nil (3 'foo) (x x)) yields an error instead of producing a valid result

### ISSUE THREAD

**Issue by funkenblatt** [VincentToups/shadchen-el#2]
Apparently when performing a match against an integer pattern, it does an (= ...) test without verifying that the match value is numeric.

**Comment by VincentToups** [comment]
funkenblatt - are you sure you are up to date?  I am just testing against the latest master and I believe the bug is already resolved (as of at least a month ago).

**Comment by funkenblatt** [comment]
Ah, I was using whichever version was in Marmalade.  Perhaps that should be updated?

**Comment by VincentToups** [comment]
Yes, it should be.  But I recall the process of updating Marmalade is pretty involved compared to a "git push".  I'll get to it eventually.

**Comment by nicferrier** [comment]
This should be closed now.
